templates/hotgraphicLayoutTiles.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function HotgraphicLayoutTiles(props) {
+
+  const {
+    _items,
+    onPinClicked
+  } = props;
+
+  return (
+    <div className="hotgraphic__tile-item-container" role="list">
+
+      {_items.map(({ _index, _graphic, _isVisited, title }) =>
+        <div className="hotgraphic__tile-item" role="listitem" key={_index}>
+
```

Comment:
Similar to accordion, add aria label in _items.map 

`const ariaLabel = ${_isVisited ? visited + '. ' : ''} ${_pin.alt ? _pin.alt : title}. Item ${(_index + 1)} of ${_items.length};`